createDataset.py

```diff
@@ -25,162 +151,74 @@ def getWhatsAppData():
-	        for counter in range(startMessageIndex, 0, -1):
-	            currentLine = allLines[counter]
-	            rightBracket = currentLine.find(']') + 2
-	            justMessage = currentLine[rightBracket:]
-	            colon = justMessage.find(':')
-	            if not currentSpeaker:
-	                # The first speaker not named me
-	                currentSpeaker = justMessage[:colon]
-	            elif (currentSpeaker != justMessage[:colon] and otherPersonsMessage):
-	                # A different person started speaking, so now I know that the first person's message is done
-	                otherPersonsMessage = cleanMessage(otherPersonsMessage)
-	                myMessage = cleanMessage(myMessage)
-	                responseDictionary[otherPersonsMessage] = myMessage
-	                break
-	            otherPersonsMessage = justMessage[colon+2:] + otherPersonsMessage
-	        myMessage, otherPersonsMessage, currentSpeaker = "","",""    
-	return responseDictionary
-
-def getLinkedInData():
-	df = pd.read_csv('Inbox.csv')
-	dateTimeConverter = lambda x: datetime.strptime(x,'%B %d, %Y, %I:%M %p')
-	responseDictionary = dict()
-	peopleContacted = df['From'].unique().tolist()
-	for person in peopleContacted:
-	    receivedMessages = df[df['From'] == person]
-	    sentMessages = df[df['To'] == person]
-	    if (len(sentMessages) == 0 or len(receivedMessages) == 0):
-	        # There was no actual conversation
-	        continue
-	    combined = pd.concat([sentMessages, receivedMessages])
-	    combined['Date'] = combined['Date'].apply(dateTimeConverter)
-	    combined = combined.sort(['Date'])
-	    otherPersonsMessage, myMessage = "",""
-	    firstMessage = True
-	    for index, row in combined.iterrows():
-	        if (row['From'] != personName):
-	            if myMessage and otherPersonsMessage:
-	                otherPersonsMessage = cleanMessage(otherPersonsMessage)
-	                myMessage = cleanMessage(myMessage)
-	                responseDictionary[otherPersonsMessage.rstrip()] = myMessage.rstrip()
-	                otherPersonsMessage, myMessage = "",""
-	            otherPersonsMessage = otherPersonsMessage + row['Content'] + " "
-	        else:
-	            if (firstMessage):
-	                firstMessage = False
-	                # Don't include if I am the person initiating the convo
-	                continue
-	            myMessage = myMessage + str(row['Content']) + " "
-	return responseDictionary
+def getDiscordData():
+    personName = raw_input('Enter your full Discord name: ')
+    # Putting all the file names in a list
+    allFiles = []
+    # Edit these file and directory names if you have them saved somewhere else
+    for filename in os.listdir('DiscordChatLogs'):
+        if filename.endswith(".txt"): 
+            allFiles.append('DiscordChatLogs/' + filename)
+    responseDictionary = dict() # The key is the other person's message, and the value is my response
```

Comment:
Nit: Same nit with the comment spacing. If it's just rendering weird on Github then its fine.